### PR TITLE
feat - made cart and wishlist page

### PR DESCRIPTION
**Pull request by therajatg**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -20,4 +20,18 @@
 .margin-one {
   margin: 3px;
 }
+  .line{
+    height: 0.5px;
+    width: 100%;
+    background-color: #121212;
+  }
```
Comment (agrawalshanu708) [inline src/App.css:27]: use colors from root

File: src/App.js [hunk 2]
```diff
@@ -1,5 +1,7 @@
 import "./App.css";
+import { Cart } from "./frontend/pages/cart/cart";
 import { Home } from "./frontend/pages/home/Home";
+import { Wishlist } from "./frontend/pages/wishlist/Wishlist";
 import Mockman from "mockman-js";
```
Comment (agrawalshanu708) [inline src/App.js:5]: try use use index for every folder, import from that, it makes code clean

**Review by agrawalshanu708 — COMMENTED** [review]
look good to merge

**Review by parulgupta2001 — APPROVED** [review]
Code is well structured. You can merge

File: src/App.css (hunk 1 above)
Comment (therajatg) [reply src/App.css:27]: will do that

File: src/App.js (hunk 2 above)
Comment (therajatg) [reply src/App.js:5]: will do that. thanks

**Pull request merged**